main/discovery_beacon.c

```diff
@@ -0,0 +1,118 @@
+#include "discovery_beacon.h"
+
+#include "driver_setup.h"
+#include "esp_log.h"
+#include "lwip/sockets.h"
+// #include "esp_netif.h"
+
+// TODO: Finish this up!!
```

Comment:
Is this finished?

Reply:
Oops, forgot to delete a temporary comment. Fixed in 72219c3.